frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js

```diff
@@ -0,0 +1,49 @@
+import React, { Fragment } from "react";
+import ApplicationDetails from "../../../../../templates/ApplicationDetails";
+
+const ProjectDetailsNavDetails = ({activeLink}) => {
+
+    // integrate once api is ready
+    const workflowDetails = {}; 
+    const { applicationDetails_ProjectDetails, applicationData_ProjectDetails } = Digit.Hooks.project.useViewProjectDetails({}); 
+    const { applicationDetails_FinancialDetails, applicationData_FinancialDetails } = Digit.Hooks.project.useViewFinancialDetails({}); 
+
+    return (
+        <>
+        {
+            (activeLink === "Project_Details") && (
+                <ApplicationDetails
+                    applicationDetails={applicationDetails_ProjectDetails}
+                    isLoading={false} //will come from backend
+                    applicationData={applicationData_ProjectDetails}
+                    moduleCode="Project"
+                    isDataLoading={false}
+                    workflowDetails={workflowDetails}
+                    showTimeLine={false}
+                    timelineStatusPrefix={""}
+                    businessService={""}
+                    forcedActionPrefix={"PROJECT"}
+              />
+            )
+        }
+        {
+            (activeLink === "Financial_Details") && (
+                <ApplicationDetails
```

Comment:
Similarly make this financial details into a pageComponent for generic purposes @hariomsinha-eGov